code/client/src/core/dev_features.cpp

```diff
@@ -46,6 +49,10 @@ namespace MafiaMP::Core {
     }
 
     void DevFeatures::Init() {
+        
+        // for 'spawn random car' feature
+        std::srand(time(NULL));
```

Comment:
Might be better if we remove this since it would affect entirety of the client only for a debug feature.

Reply:
Agreed, removed.